skrub/tests/test_minhash_encoder.py

```diff
@@ -261,3 +260,38 @@ def test_check_fitted_minhash_encoder() -> None:
     # Check that it works after fitting
     encoder.fit(X)
     encoder.transform(X)
+
+
+def test_deterministic():
+    """Test that the encoder is deterministic"""
+    # TODO: add random state to encoder
```

Comment:
Is there randomness? If it's deterministic, no need for random state.

Reply:
Right now it's deterministic because the seed in the hash is fixed, but we could imagine allowing the user to vary the seed.